ResearchKit/Localized/en.lproj/Localizable.strings

```diff
@@ -86,6 +86,10 @@
 /* Alert for invalid email address value */
 "INVALID_EMAIL_ALERT_MESSAGE" = "Invalid email address: %@";
 
+/* Alerts for invalid location coordinates */
+"LOCATION_ADDRESS" = "Enter an address";
+"LOCATION_ERROR_TITLE" = "Location error";
```

Comment:
I updated the error title to be more specific. Since this is different from an invalid value, I used "Failed to Resolve Specified Address".